### PR TITLE
App redesign

### PR DESCRIPTION
**Pull request by Prospector**

### CODE REVIEW

**Comment by imheavi** [comment]
Will there be some sort of changelog covering the redesign?

**Comment by Norbiros** [comment]
<details>
<summary>Some images of new app design:</summary>

Screenshot from 2024-11-17 15-37-44
Screenshot from 2024-11-17 15-34-26
Screenshot from 2024-11-17 15-34-09

</details>

File: packages/ui/src/components/content/ContentListPanel.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<script setup lang="ts" generic="T">
+import { ref, computed } from 'vue'
+import type { Ref } from 'vue'
+import Checkbox from '../base/Checkbox.vue'
+import ContentListItem from './ContentListItem.vue'
+import { type ContentItem } from './ContentListItem.vue'
+import { DropdownIcon } from '@modrinth/assets'
+// @ts-ignore
+import { RecycleScroller } from 'vue-virtual-scroller'
```
Comment (Norbiros) [inline packages/ui/src/components/content/ContentListPanel.vue:9]: The vue-virtual-scroller library causes a serious bug in WebKit that freezes the entire app on some devices (https://github.com/modrinth/code/issues/1127). This makes most of the app unusable, as I can’t access my projects. 

To fix it, please switch to another virtual list library like `vue-typed-virtual-list`. I’ve used it before (example: https://github.com/modrinth/code/commit/797e1f1f214011cc36112f88bde7c609237fb2e1), though it had some issues with logs not attaching to the bottom. However, this shouldn’t affect the projects list.

**Review by Norbiros — CHANGES_REQUESTED** [review]

**Comment by Prospector** [comment]
Let's not publicly post photos

**Comment by Tschipcraft** [comment]
I found this concept for redesigning the app settings page on the Labrinth Discord server by voxelfloof (Vox in a box):
https://www.figma.com/proto/Z6tildUhsoWNM7pQjW4VEO/Modrinth-Concepts?scaling=contain&content-scaling=fixed

Thought it looked really nice and could be used as inspiration

File: packages/ui/src/components/content/ContentListPanel.vue (hunk 1 above)
Comment (Geometrically) [reply packages/ui/src/components/content/ContentListPanel.vue:9]: Will fix before release

**Pull request merged**